pallets/order-book/src/lib.rs

```diff
@@ -277,23 +277,41 @@ pub mod pallet {
         },
 
         /// User placed new limit order
-        OrderPlaced {
+        LimitOrderPlaced {
             order_book_id: OrderBookId<AssetIdOf<T>>,
             dex_id: T::DEXId,
             order_id: T::OrderId,
             owner_id: T::AccountId,
         },
 
+        /// User tried to place the limit order out of the spread. The limit order is converted into a market order.
+        LimitOrderConvertedToMarketOrder {
+            order_book_id: OrderBookId<AssetIdOf<T>>,
+            dex_id: T::DEXId,
+            owner_id: T::AccountId,
+        },
+
+        /// User tried to place the limit order out of the spread.
+        /// One part of the liquidity of the limit order is converted into a market order, and the other part is placed as a limit order.
+        LimitOrderSplittedIntoMarketOrderAndLimitOrder {
```

Comment:
```suggestion
        LimitOrderSplitIntoMarketOrderAndLimitOrder {
```
just caught my eye :)